graphql_compiler/tests/test_graphql_pretty_print.py

```diff
@@ -37,9 +37,8 @@ def test_graphql_pretty_print_indentation(self) -> None:
 
     def test_filter_directive_order(self) -> None:
         bad_query = """{
-            Animal @filter(value: ["$name"], op_name: "name_or_alias") {
-                uuid @filter(value: ["$max_uuid"], op_name: "<=")
-
```

Comment:
Since these tests cover query pretty-printing, let's try to minimize or entirely eliminate changes to test inputs here. Issues like extraneous whitespace, poor formatting, directive argument order that causes poor readability, etc. are probably intentional and may in fact be precisely what the tests are testing.

Reply:
nit: It probably doesn't matter a ton, but I'd feel better about it if we didn't change anything about these tests, including just removing the blank lines.